src/routes/healthcheck/healthcheck-get.ts

```diff
@@ -10,7 +10,7 @@ export const HealthcheckGet = async (_: Request, res: Response): Promise<void> =
 		try {
 			await deepCheckCryptor();
 		} catch (e) {
-			logger.warn("Cryptor deepcheck failed.", { error: e });
+			logger.warn("Cryptor deepcheck failed in work. This happen during startup when Worker started by Cryptor sidecar is not ready.", { error: e });
```

Comment:
I'd revert back this message, I'm not sure it's helpful and there is a possibility for the sidecar to fail in production.  In which case, we don't want to give the illusion that it's just a temporary issue

Reply:
I get your concerning, but here's the reason:
1. This is line is within the `envVars.MICROS_GROUP === "Worker"`, so only `Worker` hit this line.
2. And according the `micros` settings (let me know I can further point you to the doc), `healtCheck` within `Worker` is only hit once during startup, unless we config to periodically hit worker healthcheck (which we did not).

So in short, this does only happen during worker startup.

Reply:
Then what's the benefit of updating the message?  If you don't care about it, just remove the warning?

Reply:
The benefit is I think the updated message is more clear than existing one. If anyone stumble upon this warning message, it will make it clear to them.

Not sure about removing the warning. Was thinking leave it for now since it is not `error`, not a super noise to me yet (and still maybe helpful to diagnose startup)

-------------

So I am like 6 out of 10 prefer to leave the updated message as it is. But if you are feeling stronger than a `6`, let me know, I am happy to either remove the warning message or revert the message.